packages/manager/apps/dedicated/client/app/billing/payment/method/add/routing.js

```diff
@@ -121,6 +131,9 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
           get($transition$.params(), 'from', null),
         );
       },
+
+      user: /* @ngInject */ () => Environment.getUser(),
```

Comment:
Why not use `Environment.getUser()` directly in the component/controller ? Do we expect the binding to change at one point?